Nextjs integration (#37)
last9/openapm-nodejs: README.md

openapm.instrument('nestjs');
```

### Next.js

OpenAPM supports RED metrics for both pages and app router in a Next.js application.

```js
openapm.instrument('nextjs');
```

> > Note: You can only use the library if Next.js is running in a Node.js environment. Since OpenAPM relies on prom-client for capturing metrics data a serverless environment might not be able persist them.
> 

Reviewer: ```suggestion
> Note: You can only use the library if Next.js runs in a Node.js environment. Since OpenAPM relies on prom-client for capturing metrics data, a serverless environment might be unable to persist them.

```

}
```

## API Reference

1. `instrument`: Used to instrument supported technologies. Refer the [usage][#usage] section.

> 2. `getMetrics`: Returns a Promise of string which contains metrics in prometheus exposition format. You can use this function to expose a metrics endpoint if `enableMetricsServer` is set to false. For example,
> 

Reviewer: ```suggestion
2. `getMetrics`: Returns a Promise of string which contains metrics in Prometheus exposition format. You can use this function to expose a metrics endpoint if `enableMetricsServer` is set to false. For example,

```

```js
const openapm = new OpenAPM({
  enableMetricsServer: false
});

openapm.instrument('express');

const app = express();

app.get('/metrics', async (_, res) => {
  const metrics = await openapm.getMetrics();
  res.setHeader('Content-Type', 'text/plain; version=0.0.4; charset=utf-8');
  res.end(metrics);
});
```

> 3. `shutdown`: Returns a promise which is resolved after the cleanup in OpenAPM. The cleanup includes closing the server if it has started and clearing the prom-client register.
> 

Reviewer: ```suggestion
3. `shutdown`: Returns a promise resolved after the cleanup in OpenAPM. The cleanup includes closing the metrics server if it has started and cleared the prom-client register.

```

Author: I think `clearing` is correct here.
